src/domain/i18n/locales/sv.json

```diff
@@ -9,6 +9,10 @@
   },
   "MAP": {
     "ATTRIBUTION": "OpenStreetMaps skapare",
+    "ACCESSIBILITY_NOTICE": "Karttanäkymä. Kartan tietoja voi tarkastella tällä hetkellä vain näönvaraisesti.",
```

Comment:
Finnish in swedish translation file. If you are waiting for official translation remember to change this 😄